notes-service/src/main/java/org/exoplatform/wiki/listener/AnalyticsAddTagListener.java

```diff
@@ -0,0 +1,49 @@
+package org.exoplatform.wiki.listener;
```

Comment:
Missing license Header
(To apply on all classes please to not repeat the comment multiple times)

Reply:
done